README.md

```diff
@@ -175,16 +187,33 @@ target_link_libraries(stm32-blinky-f4
 )
 ```
 
-### Building
+Here is another usage for a H7 device with the M7 core:
 
+```cmake
+target_link_libraries(${TARGET_NAME} PRIVATE
+    HAL::STM32::H7::M7::RCC
+    HAL::STM32::H7::M7::GPIO
+    HAL::STM32::H7::M7::CORTEX
+    CMSIS::STM32::H743ZI::M7
+    STM32::NoSys
+)
```

Comment:
for all "examples", shouldn't we just link to the example folder ? I mean the readme is here to explain the targets but we have dedicated examples so we should not duplicate the information here. 
@Hish15  @robamu what do you think ?

Reply:
I think it's okay if the README has a few examples as well to have a general idea how the targets are used. I added that example because it is one that uses two cores.

Reply:
Sorry for delay, we have had less time to dedicate to this repo.
In my experience duplicating too much information in the documentation often ends with documentation not up to date when the code evolves. This is why I think a link to example is better than a copy.

@Hish15  any thoughts about this ?

Reply:
Sounds good. I'll look into it

Reply:
I specifically added the dual core example to have at least one simple example for a dual core.
Maybe it would be a good idea to add the H7M7 example to the blinky application?